src/display/editor/ink.js

```diff
@@ -324,6 +335,11 @@ class InkEditor extends AnnotationEditor {
     if (this.isInEditMode() && this.currentPath.length !== 0) {
       event.stopPropagation();
       this.#endDrawing(event);
+
+      // Since the ink editor covers all the page and we
+      // want to be able to select an other editor, we
+      // just put this one in background.
```

Comment:
```suggestion
      // Since the ink editor covers all of the page and we want to be able
      // to select another editor, we just put this one in the background.
```